### PR TITLE
Info about authentication via Firebase with link to sample application

### PR DESCRIPTION
**Pull request by hpatoio**
As requested in the slack channel.

### CODE REVIEW

File: core/jwt.md [hunk 1]
```diff
@@ -161,3 +161,7 @@ default:
 ```
 
 Finally, mark your scenarios with the `@login` annotation to automatically add a valid `Authorization` header, and with `@logout` to be sure to destroy the token after this scenario.
+
+## Authenticate a Firebase user
+
+If you users login on [Firebase](https://firebase.google.com/) you can still validate the JWT in your application and thus authenticate the user. You can find a sample application with all set up info here: [hpatoio/api-platform-jwt-firebase](https://github.com/hpatoio/api-platform-jwt-firebase)
```
Comment (dunglas) [inline core/jwt.md]: your?

File: core/jwt.md [hunk 2]
```diff
@@ -161,3 +161,7 @@ default:
 ```
 
 Finally, mark your scenarios with the `@login` annotation to automatically add a valid `Authorization` header, and with `@logout` to be sure to destroy the token after this scenario.
+
+## Authenticate a Firebase user
```
Comment (dunglas) [inline core/jwt.md]: Can you change to `Authenticate a Firebase User`

Comment (hpatoio) [reply core/jwt.md]: done

File: core/jwt.md (hunk 1 above)
Comment (hpatoio) [reply core/jwt.md]: done

**Comment by hpatoio** [comment]
If it's ok I can squash and rebase.

**Pull request closed without merging**